src/info.c

```diff
@@ -344,6 +344,7 @@ void info_create(void)
     SET_DEFAULT(mode_viewer, text_bottom_left, default_viewer_bl);
     SET_DEFAULT(mode_viewer, text_bottom_right, default_viewer_br);
     SET_DEFAULT(mode_gallery, text_bottom_right, default_gallery_br);
+    SET_DEFAULT(mode_off, text_bottom_right, default_gallery_br);
```

Comment:
It is "off" mode, no any info.

Reply:
needs for status messages. i.e. "Image reloaded"
otherwise leads to crash